app/src/main/java/org/mozilla/focus/activity/MainActivity.kt

```diff
@@ -101,7 +101,8 @@ open class MainActivity : LocaleAwareAppCompatActivity() {
         window.statusBarColor = ContextCompat.getColor(this, android.R.color.transparent)
         when (resources.configuration.uiMode and Configuration.UI_MODE_NIGHT_MASK) {
             Configuration.UI_MODE_NIGHT_UNDEFINED, // We assume light here per Android doc's recommendation
-            Configuration.UI_MODE_NIGHT_NO, -> {
+            Configuration.UI_MODE_NIGHT_NO,
```

Comment:
No strong opinions here, but I think this trailing comma is not useful when matching?

Reply:
Here is just the same argument as any other trailing commas, you can add another matching value for that case without changing more than one row, preserving blame.